airbyte-webapp/src/views/Connection/ConnectionForm/__snapshots__/formConfig.test.ts.snap

```diff
@@ -2667,6 +2672,7 @@ Object {
     "normalization": "basic",
     "prefix": "",
     "scheduleData": null,
+    "scheduleType": "manual",
```

Comment:
This defaults to manual now. I'm assuming this is intended?

Reply:
I think it was also manual before. Didn't make any much changes here rather than introducing the scheduleType Field

Reply:
Didn't we already have the scheduleType field though?

Reply:
We didn't have it before this new scheduling logic